bin/busted

```diff
@@ -1,3 +1,21 @@
-#!/usr/bin/env sh
+#!/usr/bin/env perl
 
-exec resty --http-include spec/fixtures/echo.conf bin/busted.lua "$@"
+use strict;
+use warnings FATAL => 'all';
+
+use File::Basename;
+use Cwd;
+
+my $dirname = dirname(__FILE__);
+my $cwd = getcwd();
+
+chdir "$dirname/..";
+
+my $apicast = getcwd();
+
+exec '/usr/bin/env', 'resty',
```

Comment:
You are sure you want just exec Right?
 
The big difference is that system() creates a fork process and waits to see if the command succeeds or fails - returning a value. exec() does not return anything, it simply executes the command.

Reply:
Well exec replaces the current process and that is exactly what I want.

Reply:
This is just a wrapper to prepare some flags and execute different command. So to not have to handle error codes, input, output etc it is just nice to exec.